Tests/SwiftParserTest/ParserTests.swift

```diff
@@ -60,4 +73,95 @@ public class ParserTests: XCTestCase {
       name: "Self-parse tests", path: currentDir, checkDiagnostics: true
     )
   }
+
+  func testSwiftTestsuite() throws {
+    try XCTSkipIf(ProcessInfo.processInfo.environment["SKIP_SELF_PARSE"] == "1")
```

Comment:
Is this test a no-op if you don’t have `swift` checked out next to `swift-syntax`? Could you document that you need the sibling checkout to run these tests?